pipelines/broad/dna_seq/germline/joint_genotyping/JointGenotyping.wdl

```diff
@@ -1,18 +1,27 @@
 version 1.0
 
-import "../../../../../tasks/broad/JointGenotypingTasks.wdl" as Tasks
+import "../../../../tasks/JointGenotypingTasks.wdl" as Tasks
 
-
-# Joint Genotyping for hg38 Whole Genomes and Exomes (has not been tested on hg19)
 workflow JointGenotyping {
-
```

Comment:
I know this is an early PR, just wanted to mention that you'll want to keep the pipeline version and increment it, this seems like it would be a major version update to 2.0.0, as I assume the scientific outputs will be decently different.